packages/toggleContent/tests/toggleContent.test.tsx

```diff
@@ -12,6 +12,14 @@ const primary = () => <div>primary</div>;
 const secondary = () => <div>secondary</div>;
 
 describe("ToggleContent", () => {
+  beforeEach(() => {
+    const mock = jest.fn();
+    mock.mockReturnValue({
+      toString: jest.fn()
+    });
+    window.getSelection = mock;
```

Comment:
Jest doesn't support `window.getSelection`, so I had to mock it

Reply:
Makes sense.